content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -32,6 +32,9 @@ further_reading:
   - link: "https://www.datadoghq.com/blog/deploy-dotnet-core-azure-app-service/"
     tag: "Blog"
     text: "Deploy ASP.NET Core applications to Azure App Service"
+  - link: "/opentelemetry/interoperability/environment_variable_support"
+    tag: "Documentation"
+    text: "OTEL Environment Variable Configurations"
```

Comment:
Since this text is readable, I recommend using the term `OpenTelemetry` instead of `OTEL`